components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -4621,34 +4620,4 @@ public static boolean isFederatedRoleBasedAuthzEnabled(String clientId) throws I
         }
         return isFederatedRoleBasedAuthzEnabled;
     }
-
-    public static String[] getRequestedOIDCScopes(String[] requestedScopes)
```

Comment:
why are we removing a public method? Ideally, we should deprecate in the first release and then gradually we should remove it?